src/sentry/tasks/post_process.py

```diff
@@ -114,6 +116,22 @@ def _capture_event_stats(event: Event) -> None:
     metrics.timing("events.size.data", event.size, tags=tags)
 
 
+def _update_escalating_metrics(event: Event) -> None:
+    """
+    Update metrics for escalating issues when an event is processed.
+    """
+    metrics_backend = KafkaMetricsBackend()
+    metrics_backend.counter(
+        UseCaseID.ESCALATING_ISSUES,
+        org_id=event.project.organization_id,
+        project_id=event.project.id,
+        metric_name="event_ingested",
+        value=1,
```

Comment:
This should get aggregated as part of the counter right?

Reply:
yes, that's correct